core-blocks/latest-comments/index.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Server-side rendering of the `core/latest-comments` block.
+ *
+ * @package gutenberg
+ */
+
+// Used to get `_draft_or_post_title`, which prevents empty-title posts from
+// appearing with no title.
+require_once( ABSPATH . 'wp-admin/includes/template.php' );
+
+$default_comments_to_show = 5;
```

Comment:
This `$default_comments_to_show` variable is unused. I suspect it wasn't intentional to keep.

Reply:
Entirely not, sorry. Fixed.